canister/src/api/get_utxos.rs

```diff
@@ -627,6 +650,7 @@ mod test {
     }
 
     #[test]
+    #[ignore]
```

Comment:
@ielashi Is this test obsolete now?

Reply:
I think it's still useful to keep as this test is also focusing a lot on the `get_utxos` API, which isn't covered as well in `confirmation_counts.rs`.